.circleci/config.yml

```diff
@@ -25,7 +25,7 @@ commands:
 jobs:
   lint:
     docker:
-      - image: valhalla/docker:build-2.6.1
+      - image: valhalla/docker:build-3.0.1
```

Comment:
updated docker build images to include `unzip` and `spatialite-bin` as timezone building requires those and they were not available